### PR TITLE
fix EventHandlerBase removeEventListener, normalize arkit global methods, change watch and ui options

### PR DESCRIPTION
**Pull request by andyps**
@TrevorFSmith please, have a look

### CODE REVIEW

File: polyfill/platform/ARKitWrapper.js [hunk 1]
```diff
@@ -36,29 +34,29 @@ export default class ARKitWrapper extends EventHandlerBase {
...
 			this._generateGlobalCallback(callbackNames[i], i)
 		}
 
 		// Set up some named global methods that the ARKit to JS bridge uses and send out custom events when they are called
 		let eventCallbacks = [
-			['onStartRecording', ARKitWrapper.RECORD_START_EVENT],
-			['onStopRecording', ARKitWrapper.RECORD_STOP_EVENT],
-			['didMoveBackground', ARKitWrapper.DID_MOVE_BACKGROUND_EVENT],
-			['willEnterForeground', ARKitWrapper.WILL_ENTER_FOREGROUND_EVENT],
+			['arkitStartRecording', ARKitWrapper.RECORD_START_EVENT],
```
Comment (TrevorFSmith) [inline polyfill/platform/ARKitWrapper.js]: Good idea, adding the prefix.

File: polyfill/platform/ARKitWrapper.js [hunk 2]
```diff
@@ -72,21 +70,12 @@ export default class ARKitWrapper extends EventHandlerBase {
...
-					build: false,
-					plane: false,
-					warnings: true,
-					anchors: false,
-					debug: false
-				}
-			}
+			options = (options && typeof(options) == 'object') ? options : {}
+			let defaultUIOptions = {
+				browser: true
```
Comment (TrevorFSmith) [inline polyfill/platform/ARKitWrapper.js]: We should also default the recording UI to true because we want people to easily share photos and videos.

**Comment by TrevorFSmith** [comment]
Ok, other than changing the default to show the recording UI, this looks good. Once that's changed I'll merge.

Thanks, Andrey!

**Comment by andyps** [comment]
@TrevorFSmith updated

**Pull request merged**